packages/node/test/manual/release-health/test-utils.js

```diff
@@ -26,4 +26,17 @@ class BaseDummyTransport {
   }
 }
 
-module.exports = { assertSessions, constructStrippedSessionObject, BaseDummyTransport };
+function validateSessionCountFunction(sessionCounts) {
+  process.on('exit', exitCode => {
+    const { sessionCounter, expectedSessions } = sessionCounts;
```

Comment:
```suggestion
function validateSessionCountFunction({ sessionCounter, expectedSessions }) {
  process.on('exit', exitCode => {
```

Reply:
Won't work, because this way you are only passing the initial values

Reply:
Ah, right. References 🙄